README.md

```diff
@@ -19,6 +19,14 @@ $ cd git-machete
 $ sudo make install
 ```
 
+Installation could be done by following commands:
+
+```bash
+$ git clone https://github.com/VirtusLab/git-machete.git
+$ cd git-machete
+$ python setup.py install
```

Comment:
I get `ImportError: No module named setuptools`... an I guess most non-prod-Python people will get that as well :/ Pls include some hints on how to install that module in canonical way (if it requires dedicated tools like `pip` than maybe don't remove the above `make` instruction as for now)

Reply:
(I just want to make the installation as easy as possible for now coz nothing discourages people more from using any tool than installation problems :scream:)

Reply:
OK, issue with `setuptools` might be only in very agnostic python env.

I will add `pip install setuptools` and other information about python installation and tools.